mypy/main.py

```diff
@@ -809,6 +809,9 @@ def add_invertible_flag(flag: str,
     # Parse config file first, so command line can override.
     options = Options()
     parse_config_file(options, config_file, stdout, stderr)
+    # the 'dmypy start' will just ignore files in config
```

Comment:
Have you tested this? I think this might still fail.

In any case, I think the best place to do make the change here is in `dmypy_server.py` where the error message is generated.

Reply:
ok, thank you. I'll enhance the fix first.